recipes/arccommander/bld.bat

```diff
@@ -0,0 +1,8 @@
+cd %SRC_DIR%
+dotnet tool restore
+dotnet publish src\ArcCommander -c Release -p:UseAppHost=false
+set DOTNET_ROOT="%PREFIX%\lib\dotnet"
+set TOOL_ROOT="%DOTNET_ROOT%\tools\arccommander"
```

Comment:
`DOTNET_TOOLS` should already be defined for you:
https://github.com/conda-forge/dotnet-feedstock/blob/b10afdcf03e444fa5e8f89bf389be83bed3bb1af/recipe/activate.d/dotnet-root.bat#L7

You should probably make use of the pre-defined variable to avoid errors. In particular the `DOTNET_ROOT` defined here is incorrect as, on Windows, it's defined as `%CONDA_PREFIX%\dotnet`.

```suggestion
set TOOL_ROOT="%DOTNET_TOOLS%\arccommander"
```